packages/victory-core/src/index.js

```diff
@@ -9,9 +9,15 @@ export { default as Portal } from "./victory-portal/portal";
 export { default as Arc } from "./victory-primitives/arc";
 export { default as Background } from "./victory-primitives/background";
 export { default as Border, default as Box } from "./victory-primitives/border";
+export { default as Circle } from "./victory-primitives/circle";
 export { default as ClipPath } from "./victory-primitives/clip-path";
+export { default as Line } from "./victory-primitives/line";
 export { default as LineSegment } from "./victory-primitives/line-segment";
+export { default as Path } from "./victory-primitives/path";
 export { default as Point } from "./victory-primitives/point";
+export { default as Rect } from "./victory-primitives/rect";
+export { default as Text } from "./victory-primitives/text";
+export { default as TSpan } from "./victory-primitives/tspan";
```

Comment:
just grouped all the primitives together as I was figuring out what primitives were a part of box plot, happy to revert this change. I haven't investigated what the color coding represents on these exports